source/renderer/app/config/stakingStakePools.dummy.json

```diff
@@ -11,14 +11,19 @@
     "performance": 0,
     "pledge": "68",
     "pledgeAddress": "https://amira.net",
-    "producedBlocks": 82,
+    "producedBlocks": 429856,
+    "metrics": {
```

Comment:
@daniloprates I don't think we are using this anywhere? If not we should remove it.

Reply:
@daniloprates I have removed this part now.